languages/de.lang

```diff
@@ -347,6 +347,9 @@ $PALANG['broadcast_mailboxes_only'] = "Nur an Postfächer senden";
 $PALANG['broadcast_to_domains'] = "Senden an Domains:";
 
 $PALANG['pStatus_undeliverable'] = 'möglicherweise UNZUSTELLBAR ';
+$PALANG['pStatus_disabled'] = 'Konto deaktiviert ';
+$PALANG['pStatus_expired'] = 'Passwort abgelaufen ';
+$PALANG['pStatus_vacation'] = 'Abwesenheit aktiviert ';
```

Comment:
A better translation would be "Automatische Antwort aktiviert " (see the typical translations for "Vacation" - and yes, I know that the english wording isn't perfect ;-)